app/services/whatsapp/providers/whatsapp_360_dialog_service.rb

```diff
@@ -92,14 +93,27 @@ def send_attachment_message(phone_number, message)
   end
 
   def process_response(response)
-    if response.success?
+    if response.success? && response['error'].blank?
       response['messages'].first['id']
     else
-      Rails.logger.error response.body
+      handle_error(response)
       nil
     end
   end
 
+  def handle_error(response)
```

Comment:
the logic seems very similar in this class and `app/services/whatsapp/providers/whatsapp_cloud_service.rb` let see if we can move some log into the base service so that we don't need to repeat